src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1538,6 +1554,15 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='Generate and assign an Azure Active Directory Identity for this managed instance '
                    'for use with key management services like Azure KeyVault.')
 
+        c.argument('storage_account_type',
+                   arg_type=backup_storage_redundancy_param_type,
+                   options_list=['--backup-storage-redundancy', '--bsr'],
+                   help='Backup storage redundancy used to store backups')
+
+        c.argument('yes',
```

Comment:
No need to declare this argument and you just need to add `confirmation=True` for corresponding command.